src/java/org/apache/cassandra/concurrent/FutureTask.java

```diff
@@ -63,6 +77,7 @@ public void run()
         finally
         {
             call = null;
+            debuggable = null; // TODO: If we didn't do this, "debuggable" could be final. Do we actually need to?
```

Comment:
@belliottsmith I'm not entirely sure why we null out `call` here. I could avoid reasoning about threads sharing `debuggable` if it was `final`...